### PR TITLE
Add blog post about engine installation and acceptance tests

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

File: content/blog/engine-installation-and-acceptance-tests.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+Title: Development environment of Wazo-Platform and acceptance tests
+Date: 2019-11-19
+Author: The Wazo Authors
```
Comment (fredericlepied) [inline content/blog/engine-installation-and-acceptance-tests.md]: You don't assume the post? :-)

File: content/blog/engine-installation-and-acceptance-tests.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+```shell
+$ sudo apt install libvirt-daemon libvirt-client git virtinst
+```
+
+## Create a dedicated network for your Wazo-Platform
+
+This creates a isolated network, the outgoing traffic will go through my `eno1`
+network interfaces (don't forget to replace it with your own) thanks to the NAT
+automatically setuped by libvirtd.
```
[the remark this answers is not in the document]
Comment (sileht) [reply content/blog/engine-installation-and-acceptance-tests.md]: fixed

File: content/blog/engine-installation-and-acceptance-tests.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+ wazo        active   yes         yes
+
+```
+
+## Install the virtual machine
+
+```
+$ wget ...
+
+$ sudo virt-install --virt-type kvm --name wazo-acceptance --memory 2048 --disk size=20 --cdrom wazo-engine-latest-amd64.iso --os-variant debian9 --network network=wazo
```
[the remark this answers is not in the document]
Comment (sileht) [reply content/blog/engine-installation-and-acceptance-tests.md]: I updated it

File: content/blog/engine-installation-and-acceptance-tests.md (hunk 1 above)
Comment (sileht) [reply content/blog/engine-installation-and-acceptance-tests.md]: fixed

**Pull request merged**